### ISSUE TITLE
Nuconta - dados de quem recebeu pagamento/transferência

### ISSUE THREAD

**Issue by hugobarauna** [pluggyai/meu-pluggy#5]
Comecei a testar o meu pluggy e funcionou muito bem. :)

Porém, senti falta de alguns dados que não achei na resposta para do endpoint `https://api.pluggy.ai/transactions`

No aplicativo mobile no extrato da nuconta, aparece o nome da pessoa/instituição que recebeu o pagamento/transferência:

![CleanShot 2023-10-14 at 13 06 22](https://github.com/pluggyai/meu-pluggy/assets/2719/ed2f0c0e-1c2c-4e08-bdb4-e168f0757371)

Porém não estou achando isso na resposta do endpoint `https://api.pluggy.ai/transactions`

Esse dado eu conseguia antes via [Pynubank](https://github.com/andreroggeri/pynubank). E não lembro bem, mas acho que vinha via a conexão direta do Pluggy também.

É possível adicionar esse dado na API do Pluggy através do Open Finance?

**Comment by Gabrielpanga** [comment]
Olá Hugo,

Nas APIs OF esta informação não é retornada! Se a instituição, no caso o Nubank, devolver o CPF/CNPJ, vamos convertê-lo para o nome da contraparte, e ele aparecerá em `paymentData`.

Esses dados são opcionais e cada instituição os implementa como deseja.

**Comment by hugobarauna** [comment]
@Gabrielpanga obrigado.

Olhando o que vem dentro de `paymentData`, para mim aparece o nome da contraparte quando a Nubank devolve o CNPJ. Porém quando ela devolve o CPF, não vem nada na contraparte.

É esse o comportamento esperado?

**Comment by hugobarauna** [comment]
@Gabrielpanga 

> Olhando o que vem dentro de paymentData, para mim aparece o nome da contraparte quando a Nubank devolve o CNPJ. Porém quando ela devolve o CPF, não vem nada na contraparte.

> É esse o comportamento esperado?

Algum update sobre isso?

Quando tem o CPF, continua não vindo o "name" da pessoa que recebeu a transferência:

![CleanShot 2024-08-24 at 11 44 57@2x](https://github.com/user-attachments/assets/92ce33fc-27e2-4a1a-a175-1ce40959be7d)

**Comment by Gabrielpanga** [comment]
@hugobarauna Você conseguiu verificar se o problema foi resolvido?

**Comment by hugobarauna** [comment]
@Gabrielpanga 

Obrigado por perguntar. O problema continua acontecendo:

![CleanShot 2024-10-03 at 09 29 52](https://github.com/user-attachments/assets/3c0f076e-3546-4922-b1d4-7d4c70911865)

Mesmo quando tem o CPF para quem eu fiz um PIX,  o `name` vem nulo.

**Comment by Gabrielpanga** [comment]
Bom dia @hugobarauna ,

Atualmente o conector do Nubank está retornando melhor os dados. Você poderia revalidar?

**Comment by hugobarauna** [comment]
@Gabrielpanga funcionou para um CNPJ mas não para um CPF:

<img width="1724" height="896" alt="Image" src="https://github.com/user-attachments/assets/8248b269-21a4-4b4f-ae38-f0e11d87ca61" />

**Comment by Gabrielpanga** [comment]
Sim, para CPF não há obrigação de retornar o nome por questão de dados pessoais.
